bears/js/PrettierLintBear.py

```diff
@@ -0,0 +1,36 @@
+import re
+from coalib.bearlib.abstractions.Linter import linter
+from dependency_management.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='prettier',
+        use_stdout=True,
+        use_stderr=True)
+class PrettierLintBear:
+    """
+    Formats JavaScript, JSX, Flow, TypeScript, CSS, Less, SCSS, JSON, GraphQL,
+    Markdown files according to opinionated code format
+    using ``prettier``.
+    """
+    LANGUAGES = {'JavaScript', 'TypeScript', 'CSS', 'Less',
+                 'SCSS', 'Vue', 'JSON', 'GraphQL', 'Markdown'}
```

Comment:
Flow is missing, but Vue was used instead (which is itself missing in the docstring :3)

Reply:
@Makman2 
https://prettier.io/docs/en/language-support.html
Here all the supported languages are provided. JSX and Flow are Javascript Syntax so i didn't find it relevant to add them to the Languages Object. Please tell.

Reply:
Okay sounds good :) But please mention Vue then in the docstring :)